commands/request.go

```diff
@@ -334,7 +334,7 @@ func (r *request) ConvertOptions() error {
 		}
 
 		kind := reflect.TypeOf(v).Kind()
-		if kind != opt.Type() {
+		if kind != opt.Type() && opt.Type() != Strings && kind != String {
```

Comment:
Here it should be just `opt.Type() != Strings` otherwise it doesn't convert numbers.